tests/unit/test_webhook.py

```diff
@@ -489,8 +524,10 @@ def test_repo_scope_unknown(self):
         actual = helpers.repo_scope(url)
         self.assertEqual(expected, actual)
 
-    @patch('webcompat.webhooks.helpers.private_issue_moderation')
-    def test_patch_acceptable_issue(self, mock_proxy):
+    @patch.object(webcompat.webhooks.model.WebHookIssue,
+                  'moderate_private_issue')
+    @patch.object(webcompat.webhooks.model.WebHookIssue, 'close_private_issue')
+    def test_patch_acceptable_issue(self, mock_close, mock_moderate):
```

Comment:
This probably doesn't test what we want.
 
Probably just need to rewrite what is expected from the test then cover it.

Reply:
Can you provide more info?

These tests are checking the HTTP responses that `process_issue_action` returns, so probably they need to be renamed. I can do that.

![Screen Shot 2020-08-04 at 4 44 27 PM](https://user-images.githubusercontent.com/67283/89348217-c7cd7200-d671-11ea-8ed0-04cc00520372.png)

For this one, we need to mock `moderate_private_issue` and tell it to return a 200. And since it doesn't throw, it will also call `close_private_issue`. If we don't mock that out, it will complain at us. 

I think we need to add a test for `moderate_private_issue`, however.

Reply:
OK, actually, you answered my question a few comments down!